modules/imaging_uploader/README.md

```diff
@@ -1,9 +1,21 @@
-To enable the Imaging Uploader to handle large files, please update the apache configuration file
-with the following values.
+# Imaging Uploader
 
-File to update : `/etc/php5/apache2/php.ini`
+## Purpose
 
-Sample values: 
+The imaging uploader is intended to allow users to upload, browse, and track 
+pipeline insertion progress.
+
+
+## Requirements
+
+A successful scan upload should obey the following requirements: 
+
+
+### Install Configurations
+
+To enable the Imaging Uploader to handle large files, please update the apache 
+configuration file (located in `/etc/php/7.0/apache2/php.ini` for PHP 7.0) with 
+the following sample values: 
```

Comment:
This should be merged with the other configuration section and just describe what they do and why to be more consistent. (Maybe with a note saying that they likely need to be changed from the php defaults and suggesting a value.)

I think it should also just say `php.ini` and not try and guess where it's located, since it's going to vary with PHP releases, operating systems, versions, etc..

Reply:
Okay I will update based on your suggestion (I was trying to keep the initial section we had in the original Readme).